### PR TITLE
added the trigger to insert user

### PR DESCRIPTION
**Pull request by YoshinariYamanaka**
added the trigger to insert user

### CODE REVIEW

**Comment by YoshinariYamanaka** [comment]
When a user is created

Screenshot 2023-12-08 at 12 04 25 PM

Add the user who is added to `auth.users` to `public.users` as well.

Screenshot 2023-12-08 at 12 04 25 PM

The `raw_user_meta_data` column has user infos such as `username`.
 
Screenshot 2023-12-08 at 12 05 29 PM

**Comment by YoshinariYamanaka** [comment]
@samuraikun 
Could you please review this, thank you !

File: supabase/seed.sql [hunk 1]
```diff
@@ -114,3 +114,22 @@ VALUES
 (1, 1, 2, 'user1@example.com', 'https://tabi-memo/invitations/1', 'editable'),
 (1, 1, 3, 'user2@example.com', 'https://tabi-memo/invitations/2', 'editable'),
 (1, 1, 4, 'user3@example.com', 'https://tabi-memo/invitations/3', 'view_only');
+
+
+CREATE FUNCTION auth.handle_new_user()
```
Comment (samuraikun) [inline supabase/seed.sql:119]: Is that all?? Nice!! 👍

**Review by samuraikun — APPROVED** [review]
LGTM

**Comment by YoshinariYamanaka** [comment]
Thanks !

**Pull request merged**